ui/styles/theme.go

```diff
@@ -1,40 +1,68 @@
 package styles
 
-import "github.com/charmbracelet/lipgloss"
+import (
+	"github.com/charmbracelet/lipgloss"
+	"github.com/dlvhdr/gh-dash/config"
+)
 
-var indigo = lipgloss.AdaptiveColor{Light: "#5A56E0", Dark: "#383B5B"}
+var (
+	SingleRuneWidth    = 4
+	MainContentPadding = 1
+)
 
 type Theme struct {
-	MainText           lipgloss.AdaptiveColor
-	BrightMainText     lipgloss.AdaptiveColor
-	Border             lipgloss.AdaptiveColor
-	SecondaryBorder    lipgloss.AdaptiveColor
-	WarningText        lipgloss.AdaptiveColor
-	SuccessText        lipgloss.AdaptiveColor
-	FaintBorder        lipgloss.AdaptiveColor
-	FaintText          lipgloss.AdaptiveColor
-	SelectedBackground lipgloss.AdaptiveColor
-	SecondaryText      lipgloss.AdaptiveColor
-	SubleMainText      lipgloss.AdaptiveColor
+	SelectedBackground lipgloss.AdaptiveColor // config.Theme.Colors.Background.Selected
+	Border             lipgloss.AdaptiveColor // config.Theme.Colors.Border.Primary
+	FaintBorder        lipgloss.AdaptiveColor // config.Theme.Colors.Border.Faint
+	SecondaryBorder    lipgloss.AdaptiveColor // config.Theme.Colors.Border.Secondary
+	FaintText          lipgloss.AdaptiveColor // config.Theme.Colors.Text.Faint
+	MainText           lipgloss.AdaptiveColor // config.Theme.Colors.Text.Primary
+	SecondaryText      lipgloss.AdaptiveColor // config.Theme.Colors.Text.Secondary
+	SubleMainText      lipgloss.AdaptiveColor // config.Theme.Colors.Text.Inverted
+	SuccessText        lipgloss.AdaptiveColor // config.Theme.Colors.Text.Success
+	WarningText        lipgloss.AdaptiveColor // config.Theme.Colors.Text.Warning
```

Comment:
Can you rename the color names here to match with the config names?
Something like:
```go
	SelectedBackground lipgloss.AdaptiveColor // config.Theme.Colors.Background.Selected
	PrimaryBorder             lipgloss.AdaptiveColor // config.Theme.Colors.Border.Primary
	FaintBorder        lipgloss.AdaptiveColor // config.Theme.Colors.Border.Faint
	SecondaryBorder    lipgloss.AdaptiveColor // config.Theme.Colors.Border.Secondary
	FaintText          lipgloss.AdaptiveColor // config.Theme.Colors.Text.Faint
	PrimaryText           lipgloss.AdaptiveColor // config.Theme.Colors.Text.Primary
	SecondaryText      lipgloss.AdaptiveColor // config.Theme.Colors.Text.Secondary
	InvertedText      lipgloss.AdaptiveColor // config.Theme.Colors.Text.Inverted
	SuccessText        lipgloss.AdaptiveColor // config.Theme.Colors.Text.Success
	WarningText        lipgloss.AdaptiveColor // config.Theme.Colors.Text.Warning
```

Reply:
https://github.com/dlvhdr/gh-dash/pull/146/commits/ead431aa02131244dc0ba5a25ba9b30f4db15a41